### PR TITLE
Feature/axiomapy 34

### PR DESCRIPTION
**Pull request by shans-qontigo**
Updated functionalities and docs.

### CODE REVIEW

**Review by bspector-qontigo — APPROVED** [review]

**Review by wdeng-qontigo — APPROVED** [review]

File: axiomapy/axiomaapi/portfoliogroups.py [hunk 1]
```diff
@@ -136,3 +136,23 @@ def delete_portfolio_group(
...
         return response
+
+    @staticmethod
+    def patch_portfolio_groups(portfolio_group_id: int,
+                               portfolios_dict: dict,
+                               return_response: bool = False):
+        """This method patches a portfolio group based on the json provided
+
+        Args:
+            portfolio_group_id: portfolio group to be deleted
```
Comment (hzeng-qontigo) [inline axiomapy/axiomaapi/portfoliogroups.py:147]: comment looks to be inaccurate - shouldn't just be a delete op.

File: axiomapy/axiomaapi/portfoliogroups.py [hunk 2]
```diff
@@ -136,3 +136,23 @@ def delete_portfolio_group(
...
+            portfolios_dict: portfolios to upsert or remove from the portfolio group
+            return_response: If set to true, the response will be returned
+
+        Returns:
+            Success message if the portfolio group is deleted successfully. Code 204
```
Comment (hzeng-qontigo) [inline axiomapy/axiomaapi/portfoliogroups.py:152]: Again, delete is not accurate.

**Review by hzeng-qontigo — CHANGES_REQUESTED** [review]
Please can you review minor comments re. doc string? Otherwise looks good to go.  Thanks.

File: axiomapy/axiomaapi/portfoliogroups.py (hunk 2 above)
Comment (shans-qontigo) [reply axiomapy/axiomaapi/portfoliogroups.py]: Hi Huang, please find the updates committed!

**Review by hzeng-qontigo — APPROVED** [review]

**Pull request merged**